install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -3513,3 +3513,14 @@
 	showUpdateStep("Updating from 0.9.38-rc1 to 0.9.38-rc2", false);
 	updateToVersion('0.9.38-rc2');
 }
+
+if (isFroxlorVersion('0.9.38-rc2')) {
```

Comment:
do not set new versions...that's what the dbversion is for...just increase that, then set it also in lib/version.inc.php and in install/froxlor.sql (panel_settings)